patches/chrome-browser-browsing_data-chrome_browsing_data_remover_delegate.h.patch

```diff
@@ -10,17 +10,3 @@ index cffe328e0303ce8c0bb6ad94ef80331c172acd99..591d95199b6288f75ac94011b5e819d1
   private:
    using WebRtcEventLogManager = webrtc_event_logging::WebRtcEventLogManager;
  
-@@ -152,11 +153,12 @@ class ChromeBrowsingDataRemoverDelegate
-     kMediaDeviceSalts = 45,
-     // See also kDisableAutoSigninForProfilePasswords.
-     kDisableAutoSigninForAccountPasswords = 46,
-+    kIPFSCache = 47,
```

Comment:
Are we allowed to delete this? Was this value persisted anywhere?
What if add `kSomething = 47` later and it starts conflicting with `kIPFSCache`?